pkg/loader/compose/compose.go

```diff
@@ -694,6 +695,12 @@ func parseEnvironment(composeServiceConfig *types.ServiceConfig, serviceConfig *
 	}
 }
 
+func parseEnvFiles(composeServiceConfig *types.ServiceConfig, serviceConfig *kobject.ServiceConfig) {
```

Comment:
Missing a test function for this new one, if you're able to add a test for it that'd be awesome